packages/core/src/interfaces/events.ts

```diff
@@ -65,6 +65,9 @@ export enum Axis {
 	LABEL_MOUSEMOVE = 'axis-label-mousemove',
 	LABEL_CLICK = 'axis-label-click',
 	LABEL_MOUSEOUT = 'axis-label-mouseout',
+	LABEL_FOCUS = 'axis-label-focus',
+	LABEL_BLUR = 'axis-label-blur',
+	RENDER_COMPLETE = 'axis-render-complete',
```

Comment:
seems like these are added only for the new HoverableAxis components, we should also add them to the existing axis components

Reply:
I have added `RENDER_COMPLETE` event to all charts, but the `LABEL_FOCUS` && `LABEL_BLUR` events I cannot since we do not provide axis focus via tab-index on non-hoverable cartesian charts.